documentation/ntfs/ntfs_0.5.md

```diff
@@ -358,12 +358,19 @@ visual_announcement | entier (1) | Optionnel | Le véhicule dispose d'annonces v
 audible_announcement | entier (1) | Optionnel | Le véhicule dispose d'annonces sonores
 appropriate_escort | entier (1) | Optionnel | Un service d'accompagnement à bord est possible (à la montée et à la descente)
 appropriate_signage | entier (1) | Optionnel | L'affichage à bord est est claire et adapté aux personnes en déficience mentale
+school_vehicle_type | entier (2) | Optionnel | Type de transport scolaire
```

Comment:
As mentioned in my last email to pascal, "school_vehicle_type" has to be replaced by "school_vehicle"